ledger/apply/apply.go

```diff
@@ -31,6 +31,9 @@ type Balances interface {
 	// A non-nil error means the lookup is impossible (e.g., if the database doesn't have necessary state anymore)
 	Get(addr basics.Address, withPendingRewards bool) (basics.AccountData, error)
 
+	// GetEx is like Get(addr, false), but also loads specific creatable
+	GetEx(addr basics.Address, cidx basics.CreatableIndex, ctype basics.CreatableType) (basics.AccountData, error)
```

Comment:
I think that a single `Get` method would be preferable, that would have the following parameters:
Get(addr basics.Address, withPendingRewards bool, cidx basics.CreatableIndex, ctype basics.CreatableType)
where we ignore cidx of -1, and adding support for ctype of "AssetParams" or something like that.
(i.e. so that this Get call would be good for asset holding/asset params and applications )

Reply:
I want to emphasize the fact `Ex` method is more expensive.